lisa/runner.py

```diff
@@ -247,7 +256,7 @@ def _callback_completed(self, results: List[TestResult]) -> None:
         finally:
             self._results_lock.release()
 
-    def _start_loop(self) -> None:
+    def _start_loop(self) -> None:  # noqa: C901
         # in case all of runners are disabled
         runner_iterator = self._fetch_runners()
         remaining_runners: List[BaseRunner] = []
```

Comment:
The loop to fetch remaining_runners can be simple. For current logic, it needs to fetch only one. That would be enough. So change them to one line only:
```python
remaining_runners.append(next(runner_iterator))
```